tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "noImplicitOverride": true,
     "skipLibCheck": true,
     "strict": true,
+    "strictNullChecks": true,
```

Comment:
I think this is the wrong place. I believe all the polymer code is built with strict_checks = False, which triggers:

https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/defs/defs.bzl;l=100;drc=0df21fcb82f92cec7549fb19e1db43e26012988f

And, so, uses this tsconfig instead:

https://cs.opensource.google/tensorflow/tensorboard/+/master:tsconfig-lax.json

Have you been able to configure tsconfig such that the build fails without the changes from cl/474951173 and cl/474976805?

You don't have to modify this file because it already has "strict": true, which I believe already enables strictNullChecks.

https://www.typescriptlang.org/tsconfig#strict

Reply:
Thanks! Yap I was confused by not seeing build error from this tsconfig change. Didn't know the tsconfig files are separate. Will check again.